.github/workflows/analyze.yml

```diff
@@ -1,3 +1,7 @@
+# Copyright 2025 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
```

Comment:
I authored these workflow files from scratch. I don't believe you need to credit the Flutter contributors.

Reply:
It was my mistake. I've uploaded a patch. Thank you for review.